common/src/main/java/org/apache/sedona/common/geography/Functions.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.sedona.common.geography;
+
+import com.google.common.geometry.*;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.sedona.common.S2Geography.Geography;
+import org.apache.sedona.common.S2Geography.GeographyCollection;
+import org.apache.sedona.common.S2Geography.PolygonGeography;
+
+public class Functions {
+
+  public static Geography getEnvelope(Geography geography) {
```

Comment:
I suggest that we add a parameter `splitAtAntiMeridian` to control whether the lat-lng-rect was splitted at anti-meridian or not. `Geography` could naturally handle anti-meridian, the cross anti-meridian problem stems only when we treat geography objects as planar shapes, or we are exporting data in a format/spec where anti-meridian splitting is required.